core/chaincode/exectransaction_test.go

```diff
@@ -850,6 +851,10 @@ func stopChaincode(ctx context.Context, chaincodeCtx *ccprovider.CCContext, chai
 // Test the execution of a chaincode that invokes another chaincode with wrong parameters. Should receive error from
 // from the called chaincode
 func TestChaincodeInvokeChaincodeErrorCase(t *testing.T) {
+
+	_, _, stop := xtestutil.SetupNewCouchDBTestEnv()
```

Comment:
I’m concerned that these unit tests are being coupled to the couchDB extension. Is there a better way to abstract the backend in the tests?

Reply:
done